lib/routes/account.js

```diff
@@ -658,6 +638,40 @@ module.exports = function (
             )
         }
 
+        function forceTokenVerification (request, account) {
+          // If there was anything suspicious about the request,
+          // we should force token verification.
+          if (request.app.isSuspiciousRequest) {
+            return true
+          }
+          // If it's an email address used for testing etc,
+          // we should force token verification.
+          if (config.signinConfirmation) {
+            if (config.signinConfirmation.forcedEmailAddresses) {
+              if (config.signinConfirmation.forcedEmailAddresses.test(account.email)) {
```

Comment:
I see why this was done this way, to keep the line length reasonable, however, it does look a little funky to have it nested in this fashion. I have seen this in a few other spots and I know that libraries exist to solve this such as https://lodash.com/docs#get. Either way, not in the scope this PR.